common/models/service-process.json

```diff
@@ -44,6 +44,11 @@
       "default": 0,
       "description": "Watchdog timeout, in milliseconds. (0 = watchdog disabled)"
     },
+    "watchdogStallout": {
```

Comment:
No, its not a count of the number of profiles. When you do `cpu-start PROC TIMEOUT STALLOUT`, the messages goes down to the target, which echoes back a status update event: https://github.com/strongloop/strong-supervisor/blob/master/lib/targetctl.js#L103-L110

It is those isProfiling, timeout, stallout values that then go into this model.

Basically, this is so when arc lists a process, it knows if its profiling, and if it is profiling, what mode its profiling in (normal, timeout, timeout+stallout).

If you can think of some better wording to make this clear, I can update the description, but I'm fighting the data format here. Also, note line 42-46 just above: the description is identical, `/timeout/stallout/g`.